provision-app.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Include utilities.
+source provisioning-utils.sh
+
+log_step "app: Bringing up container(s)..."
+docker-compose up -d app
```

Comment:
```suggestion
docker-compose up --build --detach app
```
Because we aren't pushing this image to Docker Hub, we need to rebuild it during provisioning.